osu.Game/Graphics/Cursor/MenuCursor.cs

```diff
@@ -20,32 +20,35 @@ public class MenuCursor : CursorContainer
     {
         protected override Drawable CreateCursor() => new Cursor();
 
+        private Bindable<bool> cursorRotate;
+
         private bool dragging;
 
         private bool startRotation;
 
         protected override bool OnMouseMove(InputState state)
         {
-            if (dragging)
+            if (cursorRotate && dragging)
             {
-                Debug.Assert(state.Mouse.PositionMouseDown != null);
+                Debug.Assert (state.Mouse.PositionMouseDown != null);
 
                 // don't start rotating until we're moved a minimum distance away from the mouse down location,
                 // else it can have an annoying effect.
-                startRotation |= Vector2Extensions.Distance(state.Mouse.Position, state.Mouse.PositionMouseDown.Value) > 30;
+                startRotation |= Vector2Extensions.Distance (state.Mouse.Position, state.Mouse.PositionMouseDown.Value) > 30;
 
-                if (startRotation)
-                {
+                if (startRotation) {
                     Vector2 offset = state.Mouse.Position - state.Mouse.PositionMouseDown.Value;
-                    float degrees = (float)MathHelper.RadiansToDegrees(Math.Atan2(-offset.X, offset.Y)) + 24.3f;
+                    float degrees = (float)MathHelper.RadiansToDegrees (Math.Atan2 (-offset.X, offset.Y)) + 24.3f;
 
                     // Always rotate in the direction of least distance
                     float diff = (degrees - ActiveCursor.Rotation) % 360;
-                    if (diff < -180) diff += 360;
-                    if (diff > 180) diff -= 360;
+                    if (diff < -180)
+                        diff += 360;
+                    if (diff > 180)
+                        diff -= 360;
                     degrees = ActiveCursor.Rotation + diff;
 
-                    ActiveCursor.RotateTo(degrees, 600, Easing.OutQuint);
+                    ActiveCursor.RotateTo (degrees, 600, Easing.OutQuint);

```

Comment:
random space?

Reply:
everywhere

Reply:
MonoDevelop/Xamarin Studio inserts it ;v

Reply:
Changed code style in settings, should be ok now...